CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ project(pdfalto)
 set(CMAKE_CXX_STANDARD 11)
```

Comment:
Is this necessary for the project or just the fuzzing target? If just for the fuzzing target, I might wrap this with and `#ifdef FUZZ` or something similar.

Reply:
I reset this.